ckanext/depositar_theme/fanstatic/scss/_dataset.scss

```diff
@@ -0,0 +1,79 @@
+.dataset-heading {
+  font-size: 21px;
+}
+
+.additional-info th {
+  width: 15%;
+}
+
+.badge-inverse {
+  background-color: #cdcdcd;
+}
+
```

Comment:
```suggestion
```
We remove the custom color here to improve the visibility since the color of badge text is white now.